data/scripting/richtext.lua

```diff
@@ -36,11 +43,15 @@
 -- RST
 -- .. function:: rt(text_or_attributes[, text = nil])
 --
+--    Normally a scripts author don't have to use this function explicitly
+--    because using the tags documented here are automatically wrapped by
+--    this function.
```

Comment:
"script’s" and "doesn’t". Also the sentence’s meaning is not clear – what exactly happens when using which tags exactly and why does it imply not having to use this function?

Reply:
The previous statement was at the end of the description and said: "There is no general need to wrap an rt tag around your text,  because the backend will take care of it. So, only use this function if you wish to add some attributes to the tag."

I thought it is better to tell this right away. Suggestions to make it more understandable?

Reply:
How about:

> Usually, it is not necessary to wrap your text with this function – the appropriate tags will be added automatically if necessary. 
> Only use this function if you wish to add custom attributes to the tag, e.g. specifying a different background color for the whole area covered by this tag.

Reply:
Have tried to make it more understandable with my words. In general i think for a user it is difficult to realize that the term 'tag' and 'function' means the same here.